doc/cmake/INSTALL.md

```diff
@@ -13,20 +13,23 @@ brew install wrf-cmake -v
 For more flexibility, e.g. when changing the registry, see the manual build instructions below.
 
 ## Install dependencies
-The following libraries are required on your system to install WRF-CMake from source: [Git](https://git-scm.com/), [JasPer](https://www.ece.uvic.ca/~frodo/jasper/), [libpng](http://www.libpng.org/pub/png/libpng.html), [libjpeg](http://libjpeg.sourceforge.net/), [zlib](https://zlib.net/), [HDF5](https://support.hdfgroup.org/HDF5/), [NetCDF-C](https://www.unidata.ucar.edu/downloads/netcdf/index.jsp), [NetCDF-Fortran](https://www.unidata.ucar.edu/downloads/netcdf/index.jsp), and MPI (required if building in `dmpar` or `dm_sm` mode). The above libraries are most likely available from your system's package manager (e.g. APT, yum, Homebrew, etc.). If you do not have the latest version of these libraries installed on your system, please see [this page](LIBS.md).
+The following libraries are required on your system to install WRF-CMake from source: [CMake](https://cmake.org/), [Ninja](https://ninja-build.org/), [Git](https://git-scm.com/), [JasPer](https://www.ece.uvic.ca/~frodo/jasper/), [libpng](http://www.libpng.org/pub/png/libpng.html), [libjpeg](http://libjpeg.sourceforge.net/), [zlib](https://zlib.net/), [HDF5](https://support.hdfgroup.org/HDF5/), [NetCDF-C](https://www.unidata.ucar.edu/downloads/netcdf/index.jsp), [NetCDF-Fortran](https://www.unidata.ucar.edu/downloads/netcdf/index.jsp), and MPI (required if building in `dmpar` or `dm_sm` mode). The above libraries are most likely available from your system's package manager (e.g. APT, yum, Homebrew, etc.). If you do not have the latest version of these libraries installed on your system, please see [this page](LIBS.md).
 
 ## Build and Install WRF-CMake
 
 ### Transition from original build system
 
-| Original      | CMake          |
-| ------------- | -------------- |
-| `./configure` | `cmake ...`    |
-| `./compile`   | `make install` |
+| Original      | CMake               |
+| ------------- | ------------------- |
+| `./configure` | `cmake ..`          |
+| `./compile`   | `cmake --build .`   |
+| -             | `cmake --install .` |
 
 Further notes:
+- `cmake --install` is available from CMake version 3.15. For older versions use `cmake --build . --target install` instead.
+- The [Ninja](https://ninja-build.org/) generator needs to be specified at configure time with the `-G` option, i.e. `-GNinja`.
 - The original build system uses a series of terminal prompts when running `./configure` whereas for CMake any non-default options need to be specified as command-line arguments.
-- If you change any registry files, then just re-run `make install`.
+- If you change any registry files, then just re-run `cmake --build .`.
```

Comment:
Should be `--install`. CMake will re-build if necessary.

Reply:
This is kind of related to my comment below as I only assumed build rather than full install. Changed it to `--install`.